metaflow/plugins/kfp/kfp_cli.py

```diff
@@ -133,6 +130,12 @@ def step_init(obj, run_id, step_name, passed_in_split_indexes, task_id):
 @click.option(
     "--workflow-timeout", default=None, type=int, help="Workflow timeout in seconds."
 )
+@click.option(
+    "--workflow-kubernetes-resources-ttl",
+    default=None,
+    type=int,
+    help="Number of seconds until Argo workflows and pods are cleaned up AFTER a run finishes.",
+)
```

Comment:
This can be a metaflowconfig setting rather than a parameter, just to keep the run clean and simple, because this is an edge case and `--workflow-kubernetes-resources-ttl` is more of a global setting across runs and flows.